modules/ml/misc/java/test/MLTest.java

```diff
@@ -36,7 +36,7 @@ public void testSaveLoad() {
         String filename = OpenCVTestRunner.getTempFileName("yml");
         saved.save(filename);
         SVM loaded = SVM.load(filename);
-        assertTrue(saved.isTrained());
+        assertTrue(loaded.isTrained());
```

Comment:
it's not related ti the sample. Please rebase the branch on top of current 3.4.

Reply:
fixed